### ISSUE TITLE
Added external link icons to links in account settings

### ISSUE THREAD

**Issue by matt-west** [Automattic/wp-calypso#100334]
Links within account settings that open pages in a new tab should use the [ExternalLink](https://wordpress.github.io/gutenberg/?path=/docs/components-externallink--docs) component so they include the external arrow (↗).

<img width="503" alt="Image" src="https://github.com/user-attachments/assets/86fb772d-7caa-4ee3-bd0d-e6b32af1cbe7" />

**Comment by crisbusquets** [comment]
@jasmussen That should be `↗` instead of the icon, right?

**Comment by matt-west** [comment]
Moving this to `Ready for dev`.
